benchmark/clients/php/local.php

```diff
@@ -0,0 +1,82 @@
+<?php
+require 'vendor/autoload.php';
+
+use Google\Cloud\PubSub\V1\PublisherClient;
+use google\pubsub\v1\GetTopicRequest;
+use google\pubsub\v1\PublisherGrpcClient;
+
+function read_cert($certfile)
+{
+  $file = fopen($certfile, "r") or die ("cannot open file");
+  $content = fread($file, filesize($certfile));
+  fclose($file);
+  return $content;
+}
+
+function grpc_get($host, $port, $creds)
+{
+  $client = new PublisherGrpcClient(sprintf("%s:%d", $host, $port), [
+    'credentials' => $creds,
+  ]);
+  $meta = array('x-goog-api-client' => ['gl-php/5.5.9-1ubuntu4.21 gapic/0.5.1 gax/0.9.0 grpc/0.1.0']);
+
+  return function($top) use($client, $meta) {
+    $req = new GetTopicRequest();
+    $req->setTopic("foo");
+    list($resp, $status) = $client->getTopic($req, $meta)->wait();
+    return $resp;
+  };
+}
+
+function vkit_get($host, $port, $creds)
+{
+  $client = new PublisherClient([
+    'serviceAddress' => $host,
+    'port' => $port,
+    'sslCreds' => $creds,
+  ]);
+  return function($top) use($client) {
+    return $client->getTopic($top);
+  };
+}
```

Comment:
I assume you mean from the GAPIC client? If you are using the grpc stub, you can make a pseudo-async call just by calling the appropriate stub method, which returns a call object with a `wait()` method. The call will run in the background, and then you call `wait()` when you need the response. I say pseudo-async because your only option is to call `wait()`, which is blocking - you cannot check for completion without blocking.